### PR TITLE
docs: update hello world doc

### PR DESCRIPTION
**Pull request by CatStark**

### CODE REVIEW

File: chapters/helloworld/index.rst [hunk 1]
```diff
@@ -2,8 +2,18 @@
 Jina "Hello, World!" 👋🌍
 *************************
 
+.. meta::
+   :description: Jina "Hello, World!"s
+   :keywords: Jina, hello world
 
-As a starter, we invite you to try Jina's **Hello, World** - a simple demo of image neural search for `Fashion-MNIST <https://hanxiao.io/2018/09/28/Fashion-MNIST-Year-In-Review/>`_. You will see how to index and query the downloaded dataset. And you will see the retrieved results in a new webpage.
+.. note:: This guide expects you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://docs.jina.ai/chapters/101/index.html>`_ first.
```
Comment (alexcg1) [inline chapters/helloworld/index.rst:9]: We're saying this in multiple documents (e.g. 102, some tutorials). Perhaps we should have some standard boilerplate snippet folks can copy/paste (or even embed if the docs platform supports it)

Means less rooms for typos, ensures link consistent, eases maintenance

BTW, we often use http://101.jina.ai at the moment

File: chapters/helloworld/index.rst [hunk 2]
```diff
@@ -41,7 +51,11 @@ On Linux:
    :align: center
 
 
-With this script you will download the **Fashion-MNIST** training and test data. Once you have the data, Jina will *index* 60,000 images from the training set. And will use random sample images from the test set as *queries*. Jina will then retrieve the relevant results. After around 1 minute, a web page will open and show results like this:
+Overview
+====================
+
+
+With this script you will download the **Fashion-MNIST** training and test data. Once you have the data, Jina will :term:`index<indexing>` 60,000 images from the training set. And will use random images from the test set as *queries*. Jina will then retrieve the relevant results. After around 1 minute, a web page will open and show results like this:
```
Comment (alexcg1) [inline chapters/helloworld/index.rst]: Should we convert to numbered list? Each sentence is very short and describes one step

File: chapters/helloworld/index.rst [hunk 3]
```diff
@@ -82,7 +96,7 @@ And the implementation behind it? It's simple:
     .. image:: hello-world-flow.png
        :align: center
 
-All the big words you can name: computer vision, neural IR, microservice, message queue, elastic, replicas, and shards all happened in just one minute!
+All the big words you can name: computer vision, :term:`neural search, microservice, message queue, elastic, and :term:`shards` all happened in just one minute!
```
Comment (alexcg1) [inline chapters/helloworld/index.rst]: What does elastic mean in this context? It only makes me think of Elastic Search. Perhaps we should include in glossary and link accordingly

@hanxiao ?

File: chapters/helloworld/index.rst (hunk 1 above)
Comment (alexcg1) [reply chapters/helloworld/index.rst:9]: @bwanglzu does the platform support embeds?

**Review by alexcg1 — COMMENTED** [review]
Nice :+1:  Some minor feedback and questions

File: chapters/helloworld/index.rst (hunk 3 above)
Comment (CatStark) [reply chapters/helloworld/index.rst]: I was also going to ask about this. I wasn't sure what is this about, it was on the previous Hello-world

File: chapters/helloworld/index.rst (hunk 2 above)
Comment (CatStark) [reply chapters/helloworld/index.rst]: Done

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Comment by alexcg1** [comment]
Do we need @FionnD to check too?

**Comment by CatStark** [comment]
> Do we need @FionnD to check too?

I checked with @FionnD and he said that your review is enough.

**Pull request merged**